relay/api/exchange/resources.py

```diff
@@ -55,7 +55,36 @@ def get(self, args):
         }
 
 
+class OrderDetail(Resource):
+
+    def __init__(self, trustlines: TrustlinesRelay) -> None:
+        self.trustlines = trustlines
+
+    args = {
+        'includeFilled': webfields.Boolean(required=False, missing=False),
+        'includeCancelled': webfields.Boolean(required=False, missing=False),
+        'includeUnavailable': webfields.Boolean(required=False, missing=False)
+    }
+
+    @use_args(args)
+    def get(self, args, order_hash: str):
+        order_orm = self.trustlines.orderbook.get_order_by_hash(bytes.fromhex(order_hash[2:]))
+
+        if order_orm is None:
+            abort(422, message='Order does not exist')
+
+        order = order_as_dict(order_orm)
+        if (args['includeFilled']):
```

Comment:
the available amount should be in there already https://github.com/trustlines-network/relay/blob/33bde27ea4eec43c4d89c1473d3fb526d753b267/relay/exchange/exchange_db.py#L69. 
Or is there something wrong with the exsiting code?